security.md

```diff
@@ -71,6 +72,15 @@ All security advisories made for Foreman are listed below with their correspondi
 
 ### Disclosure details
 
+#### <a id="2017-7535"></a>CVE-2017-7535: Stored XSS when assigning unassigned hosts to organization/location
+
+An organization or location named e.g. "<script>do_evil();</script>" may cause the script to be executed if a user tries to assign selected or all unassigned hosts to said organization or location.
+
+This issue was reported by Sanket Jagtap.
+
+* Affects Foreman 1.1 and higher.
+* Redmine issue [#20963](http://projects.theforeman.org/issues/20963)
```

Comment:
could you please add release info (1.16)? or do you want me to merge it and we'll remember to update it when 1.16 is out?